COMPATIBILITY.md

```diff
@@ -0,0 +1,18 @@
+# React Native and Expo compatibility
+
+
+| Realm JavaScript       | React Native     | Expo     | Hermes | npm    | node   |
+|------------------------|------------------|----------|--------|--------|--------|
+| 11.0.0-rc.2            | 0.70             | 46       | [x]    | >= 7   | >= 13  |
```

Comment:
According to https://blog.expo.dev/expo-sdk-46-beta-is-now-available-9dfee4040aa7 the 46 release of Expo includes only 0.69.1 and I don't see 0.70.0 mentioned in [their changelog](https://github.com/expo/expo/blob/main/CHANGELOG.md) either.